microsoft-edge/web-platform/user-agent-guidance.md

```diff
@@ -26,15 +28,15 @@ This article describes the methods Microsoft Edge supports for retrieving user a
 
 ## Feature detection
 
-Microsoft recommends detecting if a feature is supported in your browser. We do not recommend that you use feature detection to detect a specific browser. For more information see [feature detection][MdnLearnToolsTestingCrossBrowserTestingFeatureDetection].
+Microsoft recommends detecting if a feature is supported in your browser. For more information, see [feature detection][MdnLearnToolsTestingCrossBrowserTestingFeatureDetection].

```

Comment:
Could we update this to say that we prefer feature detection over any form of browser detection whenever possible?

Ex:

> Microsoft recommends detecting if a feature is supported in your browser whenever possible instead of detecting the browser.